dashboard/src/components/Layout/Clarity.tsx

```diff
@@ -37,35 +39,36 @@ import "@clr/ui/clr-ui.min.css"; // light clarity UI theme
 
 Icons.addIcons(
```

Comment:
Just sorting alphabetically the icons and adding the missing `windowCloseIcon` one.